setup.py

```diff
@@ -53,7 +53,7 @@
     tests_require=tests_require,
     install_requires=[
         "openeo>=0.15.0",
-        "openeo_driver>=0.78.0.dev",
+        "openeo_driver==0.80.0a1_block_large_requests",
```

Comment:
I don't think this is something we want to merge in master

Reply:
This will be removed during merging. It was an attempt to not interfere with the main branch.
Actually I can remove it now, as it did not work...

Reply:
FFR: https://peps.python.org/pep-0440/#pre-release-separators